.travis.yml

```diff
@@ -1,37 +0,0 @@
-language: c
-
-addons:
-  apt_packages:
-    - libgmp-dev
-    - libreadline-dev
-    - zlib1g-dev
-    - libczmq-dev
-
-matrix:
-  include:
-    - env: GAPBRANCH=master
-    - env: GAPBRANCH=stable-4.11
-    - env: GAPBRANCH=stable-4.10
-    - env: GAPBRANCH=stable-4.9
-    - env: GAPBRANCH=master ABI=32
-      addons:
-        apt_packages:
-          - libgmp-dev:i386
-          - libreadline-dev:i386
-          - gcc-multilib
-          - g++-multilib
-          - libczmq-dev:i386
```

Comment:
For the recorded, I consciously dropped the 32 bit test, as there were issues installing libczmq-dev:i386 and I decided it's not worth the bother to figure those out.